packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -10,6 +10,18 @@ export 'package:file_selector_platform_interface/file_selector_platform_interfac
     show XFile, XTypeGroup;
 
 /// Open file dialog for loading files and return a file path
+///
+/// The [acceptedTypeGroups] argument is the file type that can be selected in the dialog.
+/// When omitted, Open file dialog with all file types.
+///
+/// The [initialDirectory] argument is directory that will be displayed when the dialog is opened.
+/// (NOTICE: specify a directory as a full path, not a relative path.)
```

Comment:
The parenthetical seems like a needlessly verbose construction here. "[initialDirectory] is the full path to the directory that will be displayed when the dialog is opened." conveys the same information.